src/operators/mergeOperators.ts

```diff
@@ -0,0 +1,44 @@
+import { OperatorFunction, merge, pipe } from 'rxjs';
+import { share } from 'rxjs/operators';
+
+/**
+ * Runs operators in parallel and merges their results
+ *
+ * For each operator, the returned observable is subscribed to a pipe from the
+ * source observable with the operator. This makes it a bit like the `flatMap`
+ * operator and the `merge` function, but on an operator level instead of value
+ * or observable level.
+ *
+ * NB: Each operator will create a "copy" of the stream, so any operators
+ *     before the `coldMergeOperators` operator, will be executed for each
+ *     operator passed to `coldMergeOperators`. Because of this, you might want
+ *     to use the `mergeOperators` operator, which includes a `share` operator
+ *     to make the upstream hot.
+ *
+ * @param operators Operators to run in parallell and merge the results of
+ */
+export const coldMergeOperators = <T, R>(
```

Comment:
Is there a good reason for the `coldMergeOperators`? Would you not always prefer to use `mergeOperators` instead?

Reply:
Correct. `coldMergeOperators` is therefore only exposed through `rxbeach/internal`. 
The idea was to have smaller units to test.
It would probably be just as fine to inline it, but I don't see a problem keeping them separate.